src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -389,6 +415,18 @@
     display: flex;
     align-items: center;
   }
+
+  &.pf-m-text {
```

Comment:
i wonder if you need pf-m-text or if you could select the classes that you added for the text inside

Reply:
Good question. I put it here because the classes within (`pf-c-page__header-text`, `pf-c-page__header-label`) actually belong to `.pf-c-page` component and may not be present at all if a single string without a wrapper is used. `.pf-m-text` belongs to `pf-c-drowpdown` and `page__label` and `page__text` belong to page as they are unique.

It can be used like this as well
```
<div class="pf-c-dropdown__menu-item pf-m-text">
  string
</div>
```

Reply:
okay. could this modifier be called something like `pf-m-account-info` , text might be a bit arbitrary